resources/js/i18n/de.json

```diff
@@ -761,16 +769,39 @@
         },
         "importer": {
             "title": "Daten-Importer",
+            "entity_settings": "Entitätseinstellungen",
+            "attribute_mapping": "Attribut-Zuordnung",
             "upload_csv_info": "Wähle eine CSV-Datei aus um Daten zum importieren.",
             "drop_csv_file": "CSV-Datei hier ablegen",
             "selected_entity_type": "Entitätstyp",
             "column_entity_name": "Entitätsname-Spalte",
             "column_entity_parent": "Übergeordnete Entität-Spalte",
             "import_btn": "Starte Datenimport",
+            "validate": "Validieren",
             "no_missing_values": "Keine fehlenden Werte. Du kannst loslegen!",
             "missing_required_values": "{miss} von 1 Eintrag ist leer! Diese Option erlaubt keine leeren Einträge. Bitte ändere das. | {miss} von {total} Einträgen sind leer! Diese Option erlaubt keine leeren Einträge. Bitte ändere das.",
             "missing_non_required_values": "{miss} von 1 Eintrag ist leer! Ausgefüllte Einträge sind für diese Option nicht notwendig, aber du möchtest sie vielleicht nochmal überprüfen. | {miss} von {total} Einträgen sind leer! Ausgefüllte Einträge sind für diese Option nicht notwendig, aber du möchtest sie vielleicht nochmal überprüfen.",
-            "success": "Import von 1 Eintrag abgeschlossen | Import von {cnt} Einträgen abgeschlossen"
+            "success": "Import von 1 Eintrag abgeschlossen | Import von {cnt} Einträgen abgeschlossen",
+            "validation": {
+                "title": "Validierung",
+                "mixed": "Einige Entitäten existieren bereits und werden aktualisieren, die restlichen werden neu erzeugt!",
+                "create": "Keine Überschneidung, alle Entitäten werden neu erzeugt!",
+                "update": "Alle Entitäten existieren bereits und werden aktualisiert!",
+                "multiple_error": "Manche Pfade existieren mehrfach, daher kann der Import nicht ausgeführt werden!",
+                "no_items": "Es gibt keine Werte zum importieren!"
+            },
+            "indicator": {
+                "missing_value": "Kein Wert gesetzt",
+                "ok": "Dieser Wert ist korrekt"
+            },
```

Comment:
More info for the user would be helpful. That's why I used `no_missing_values`, `missing_required_values` and `missing_non_required_values` as keys.

Reply:
Fixed in e1f017ea68742e1c0d73bf36848c7e753edfb3c9